### PR TITLE
[Controller] add change tap_mode func

### PR DESCRIPTION
**Pull request by jin-xiaofeng**
### This PR is for:
- Server

### CODE REVIEW

File: server/controller/service/vtap.go [hunk 1]
```diff
@@ -743,3 +753,112 @@ func VTapRebalance(args map[string]interface{}) (*model.VTapRebalanceResult, err
...
+
+	if len(updateMap) == 0 {
+		return fmt.Errorf("the vtap(name=%s type=%d) failed to modify %d tap_mode",
+			vtap.Name, vtap.Type, tapMode)
+	}
+
+	return mysql.Db.Model(&vtap).Updates(updateMap).Error
+}
+
+func concurrentUpdateVTapTapModel(lcuuid string, tapMode int, response chan string, wg *sync.WaitGroup) {
```
Comment (SongZhen0704) [inline server/controller/service/vtap.go:829]: TapMode

File: server/controller/router/vtap.go [hunk 2]
```diff
@@ -42,6 +42,7 @@ func VtapRouter(e *gin.Engine) {
 
 	e.PATCH("/v1/vtaps-license-type/:lcuuid/", updateVtapLicenseType)
 	e.PATCH("/v1/vtaps-license-type/", batchUpdateVtapLicenseType)
+	e.PATCH("/v1/vtaps-tap-model/", batchUpdateVtapTapModel)
```
Comment (SongZhen0704) [inline server/controller/router/vtap.go:45]: vtaps-tap-mode

**Review by SongZhen0704 — DISMISSED** [review]

File: server/controller/service/vtap.go (hunk 1 above)
Comment (jin-xiaofeng) [reply server/controller/service/vtap.go]: done

File: server/controller/router/vtap.go (hunk 2 above)
Comment (jin-xiaofeng) [reply server/controller/router/vtap.go]: done

**Review by SongZhen0704 — APPROVED** [review]

**Pull request merged**